installing/installing_aws/preparing-to-install-on-aws.adoc

```diff
@@ -0,0 +1,49 @@
+[id="preparing-to-install-on-aws"]
+= Preparing to install on AWS
+include::modules/common-attributes.adoc[]
+:context: preparing-to-install-on-aws
+
+toc::[]
+
+[id="preparing-to-install-on-aws-prerequisites"]
+== Prerequisites
+
+* You reviewed details about the xref:../../architecture/architecture-installation.adoc#architecture-installation[{product-title} installation and update] processes.
+
+* You read the documentation on xref:../../installing/installing-preparing.adoc#installing-preparing[selecting a cluster installation method and preparing it for users].
+
+[id="requirements-for-installing-ocp-on-aws"]
+== Requirements for installing {product-title} on AWS
+
+Before installing {product-title} on Amazon Web Services (AWS), you must create an AWS account. See xref:../../installing/installing_aws/installing-aws-account.adoc#installing-aws-account[Configuring an AWS account] for details about configuring an account, account limits, account permissions, IAM user setup, and supported AWS regions.
+
+See xref:../../installing/installing_aws/manually-creating-iam.adoc#manually-creating-iam-aws[Manually creating IAM for AWS] for the options that are available if the cloud identity and access management (IAM) APIs are not accessible in your environment, or if you do not want to store an administrator-level credential secret in the `kube-system` namespace.
```

Comment:
This was a little confusing for me to follow, but might be smoother if rearranged. For your consideration:

> If the cloud identity and access management (IAM) APIs are not accessible in your environment, or if you do not want to store an administrator-level credential secret in the `kube-system` namespace, see _Manually creating IAM for AWS_ for other options.

Reply:
@jeana-redhat thank you for your review and for highlighting this. I think your suggestion is much clearer and I will use it in this PR. Thanks for the example!